tests/middlewares/signupValidator.test.js

```diff
@@ -0,0 +1,56 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../../src';
+
+chai.use(chaiHttp);
+chai.should();
+const fakeUser = {
+  userName: 'dgijh',
+  email: 'murediana@gmail.com',
+  password: 'diane3456',
+  confirmPassword: 'diane3456'
+};
+
+describe('Testing the signup validator', () => {
+  it('should return an error if the email is invalid', (done) => {
+    const { email, ...rest } = fakeUser;
+    chai
+      .request(app)
+      .post('/api/auth/signup')
+      .send(rest)
+      .end((err, res) => {
+        const { status, body } = res;
+        expect(status).to.equal(400);
+        expect(body).to.have.property('message');
```

Comment:
what is the expected value?? specially for message

Reply:
@murediane please address @alainburindi's feedback.